javascript/ql/src/Performance/PolynomialReDoS.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Polynomial regular expression used on uncontrolled data
+ * @description A regular expression that can require polynomial time
+ *              to match user-provided values may be
+ *              vulnerable to denial-of-service attacks.
```

Comment:
Suggesting changing capitalization (Denial-of-Service) but unsure whether it really matters.

Reply:
Skipping this suggestion (for now), in the name of consistency with the other 20 occurrences of `denial-of-service`.